products/ocp4/profiles/pci-dss-node.profile

```diff
@@ -20,6 +20,7 @@ filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "o
 
 # Req-2.2
 extends: cis-node
+extends: pci-dss-node-3.2.1
 
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.